package/kedro_viz/models/graph.py

```diff
@@ -528,6 +526,22 @@ def __post_init__(self, data_node: DataNode):
         if not data_node.is_free_input:
             self.run_command = f'kedro run --to-outputs="{data_node.full_name}"'
 
+    @staticmethod
+    def load_latest_metrics_data(dataset: Any) -> Optional[Dict[Any, Any]]:
+        """Load data for latest versions of the metrics dataset
+        Args:
+            dataset: the latest version of the metrics dataset
+        Returns:
+            A dictionary containing json data for the latest version
+        """
+        pattern = str(dataset._get_versioned_path("*"))

```

Comment:
Can you put a comment here linking to the place on kedro core where the cache happens and explains that we have to do this, not relying on core's native API because it has a cache?